programs/survival/compass.sc

```diff
@@ -0,0 +1,611 @@
+__config()->{
+    'stay_loaded' -> true,
+    'commands' -> {
+        'giveCompass' -> _() -> giveCompass(entity_selector('@p'): 0),
```

Comment:
wonder why using entity selector to select a player rather than just `player()`

Reply:
At the time, I didn't really trust the player() function to give the player that actually ran the command, no idea why I thought that